qutebrowser/browser/urlmarks.py

```diff
@@ -91,7 +91,7 @@ def __init__(self, parent=None):
         self.marks = collections.OrderedDict()
         self._lineparser = None
 
-        if standarddir.config() is None:
+        if standarddir.data() is None:
```

Comment:
This condition used to do what the `UrlMarkManager`'s docstring says:
```
        _lineparser: The LineParser used for the marks, or None
                     (when qutebrowser is started with -c '').
```
What happens now if you start `qutebrowser -c ""`? `_lineparser` would not be `None` and I guess there would be bad side effects.

Not to mention that the whole condition belongs to `_init_lineparser` and the constructor should check if `self._lineparser` is still `None` after calling `_init_lineparser`. Somebody should do some refactoring here... But as that code is not your fault, I apologize for bringing it up here. Please feel free to take care of this or leave it, I can create a separate issue/PR later.